src/docs/product/data-management-settings/filtering/index.mdx

```diff
@@ -24,8 +24,24 @@ Inbound filters include:
 - From specific release versions of your code
 - From certain IP addresses
 
+<Note>
+
+Filters do not apply to [sessions](/product/releases/health/#session). Error messages from minidumps also do not yet apply.
+
+</Note>
+
 Once applied, you can track the filtered events (numbers and cause) using the graph provided at the top of the Inbound Data Filters view.
 
 ![](builtin-inbound-filters.png)
 
-Data filters do not apply to [sessions](/product/releases/health/#session).
+## How Inbound Filters Work
+
+Inbound data filters are not case sensitive.
+
+### Filtering by Error Message
```

Comment:
Once we add more sections these will all read a redundant prefix "Filtering by ...". Is that intended?

Reply:
I will fix. Thanks.